### ISSUE TITLE
Problems with en.yaml

### ISSUE THREAD

**Issue by s22-tech** [leotiger/grav-plugin-backup-manager#20]
FYI, en.yaml has 2 duplicated keys, which completely borks Grav. 😱

```
BACKUP_TYPE: Backup Scope
BACKUP_TYPE: Type
```
and
```
BACKUP_DEFAULTS: Defaults
BACKUP_DEFAULTS: Defaults
```

Removing the duplicate for each of these solves the problem. 😀

**Comment by leotiger** [comment]
Thx. Fixed. Plugin won't receive more updates... Will publish the last update with including this fix.

**Comment by leotiger** [comment]
Fixed in version v0.1.7
